### PR TITLE
Alerting: Store sensitive settings encrypted for OpsGenie notifier

### PR DESCRIPTION
**Pull request by wbrowne**
**What this PR does / why we need it**:
We need to update all alert notifiers so that they support storing sensitive information encrypted. This PR covers OpsGenie 

**Which issue(s) this PR fixes**:
(part of) https://github.com/grafana/grafana/issues/25967

### CODE REVIEW

File: pkg/services/alerting/notifiers/opsgenie.go [hunk 1]
```diff
@@ -21,9 +21,22 @@ func init() {
 		OptionsTemplate: `
       <h3 class="page-heading">OpsGenie settings</h3>
       <div class="gf-form">
-        <span class="gf-form-label width-14">API Key</span>
-        <input type="text" required class="gf-form-input max-width-22" ng-model="ctrl.model.settings.apiKey" placeholder="OpsGenie API Key"></input>
-      </div>
+        <label class="gf-form-label max-width-14">API Key</label>
+		<div class="gf-form gf-form--grow" ng-if="!ctrl.model.secureFields.apiKey">
```
Comment (aknuds1) [inline pkg/services/alerting/notifiers/opsgenie.go:25]: The indentation is wrong here isn't it?

File: pkg/services/alerting/notifiers/opsgenie.go [hunk 2]
```diff
@@ -21,9 +21,22 @@ func init() {
...
+		  <input type="text"
+              required
+              class="gf-form-input max-width-22"
```
Comment (aknuds1) [inline pkg/services/alerting/notifiers/opsgenie.go:28]: Indentation it looks like.

**Review by aknuds1 — APPROVED** [review]
LGTM!

**Comment by aknuds1** [comment]
What's the milestone, 7.2?

**Review by marefr — APPROVED** [review]
Great. LGTM

**Review by marefr — CHANGES_REQUESTED** [review]
Oops sorry. I forgot one thing. Documentation. We've added a new column to denote what secure settings are supported here: https://grafana.com/docs/grafana/latest/administration/provisioning/#supported-settings

If you could update this section that would be awesome: https://github.com/grafana/grafana/blob/master/docs/sources/administration/provisioning.md#alert-notification-opsgenie

**Review by oddlittlebird — APPROVED** [review]
LGTM

**Review by marefr — APPROVED** [review]
LGTM :+1:

**Pull request merged**